backend/service/github/github.go

```diff
@@ -273,3 +275,21 @@ func (s *svc) GetFile(ctx context.Context, ref *RemoteRef, path string) (*File,
 
 	return f, nil
 }
+
+func (s *svc) CompareCommits(ctx context.Context, ref *RemoteRef, compareSHA string) (*scgithubv1.CommitComparison, error) {
+	cc := &scgithubv1.CommitComparison{
+		Status: scgithubv1.CommitCompareStatus_UNKNOWN,
+	}
```

Comment:
any reason to declare and return this instead of using `nil` when err?

Reply:
nil's more straightforward than what I had done, https://github.com/lyft/clutch/pull/278/commits/c01e8d5db549e8f348539ae37711a59372a7b5c2 cleans this up.